src/rebuild/srv.c

```diff
@@ -1181,29 +1183,21 @@ rebuild_task_ult(void *arg)
 	iv.riv_obj_count	= rgt->rgt_status.rs_obj_nr;
 	iv.riv_rec_count	= rgt->rgt_status.rs_rec_nr;
 	iv.riv_size		= rgt->rgt_status.rs_size;
+	iv.riv_seconds          = rgt->rgt_status.rs_seconds;
 
 	rc = rebuild_iv_update(pool->sp_iv_ns,
 			       &iv, CRT_IV_SHORTCUT_NONE,
 			       CRT_IV_SYNC_LAZY);
+	if (rc)
+		D_ERROR("rebuild_iv final update fails"DF_UUID": rc %d\n",
+			DP_UUID(task->dst_pool_uuid), rc);
 
-	if (rgt) {
-		int rc1;
-
-		/* Update the rebuild status, so query can get the rebuild
-		 * status.
-		 */
-		rgt->rgt_status.rs_version = rgt->rgt_rebuild_ver;
```

Comment:
Why remove this? It is checked inside the following rebuild_status_completed_update().

Reply:
it is already set in rebuild_global_pool_tracker_create